standard/standard_2.php

```diff
@@ -949,7 +949,10 @@ function rand(int $min = 0, int $max): int {}
  * </p>
  * @return void
  */
-function srand(int $seed, int $mode = MT_RAND_MT19937): void {}
+function srand(
+    int $seed,
+    #[PhpStormStubsElementAvailable(from: '7.1')] int $mode = MT_RAND_MT19937
```

Comment:
Even though it [says](https://www.php.net/manual/en/function.srand.php) `$mode = MT_RAND_MT19937`, it returns `0` as a default value: https://3v4l.org/t9nVj#v800
Looks like a documentation issue?
Also: `int` parameter type appears since 8.0 only.

Reply:
Reflection returns numeric value of `define("MT_RAND_MT19937", 0);`
`int` as a typehint appears since 8.0 because generally typehints were introduced in reflection in php 8.0. As far as this is `function` but not `method` it can't be overridden and thus there will be no problems with hierarchy checks so typehint here is ok.